lib/services/screen-builder-service.ts

```diff
@@ -5,18 +5,24 @@ import * as path from "path";
 import * as util from "util";
 
 export class ScreenBuilderService implements IScreenBuilderService {
+	private static SCREEN_BUILDER_SPECIFIC_FILES = [".yo-rc.json", ".app.json", "app.js"];
+	private static UPGRADE_ERROR_MESSAGE = 'Your app has been build with an obsolete version of Screen Builder. Please, migrate it to latest version and retry.';
```

Comment:
Noted and reflected in the message shown on the console, instead of this one. That message being 4 rows below in the `private static UPGRADE_ERROR_MESSAGE_SHOWN_ON_THE_CONSOLE` variable